setup.py

```diff
@@ -30,7 +30,7 @@ def get_long_description():
     license_file="LICENSE",
     python_requires=">=3.6.0",
     packages=["unimport"],
-    install_requires=["libcst==0.3.10"],
+    install_requires=["libcst==0.3.10", "pathspec==0.8.0"],
```

Comment:
Let's add this library to the extras_require section.
People who do not want to use this feature should not have to download this library.